src/main/java/net/neoforged/neoforge/network/HandshakeHandler.java

```diff
@@ -269,24 +270,26 @@ void handleRegistryMessage(final HandshakeMessages.S2CRegistry registryPacket, f
     private boolean handleRegistryLoading(final NetworkEvent.Context contextSupplier) {
         // We use a countdown latch to suspend the impl thread pending the client thread processing the registry data
```

Comment:
This comment is outdated now since there's no more countdown latch being used

Reply:
was wondering if anyone would bring it up 😇 fixed!